jbpm-human-task/jbpm-human-task-audit/src/test/java/org/jbpm/services/task/audit/service/TaskAuditBaseTest.java

```diff
@@ -319,45 +321,65 @@ private void testDescriptionUpdate(String oldDescription, String newDescription,
         List<TaskEvent> taskEvents = taskAuditService.getAllTaskEvents(taskId, new QueryFilter());
         if (changeExpected) {
             Assertions.assertThat(taskEvents).hasSize(2);
-            Assertions.assertThat(taskEvents.get(1).getMessage()).contains(String.valueOf(oldDescription),
-                    String.valueOf(newDescription));
+            assertEquals(expectedMessage,
+                         taskEvents.get(1).getMessage());
```

Comment:
I would not mix jUnit Asserts with AssertJ. Would you mind changing it back to AssertJ, please? `Assertions.assertThat(taskEvents.get(1).getMessage()).isEqualTo(expectedMessage);` And in other places too, please. Thanks.

Reply:
@MarianMacik in this test there is a lot of assertEquals applied, you mean change that on the ones that my changes introduced, right?

Reply:
Sure. No need to rewrite whole class :) In general we try to use AssertJ in newly added tests as it not only provides more methods but also it is easier to read.